app.js

```diff
@@ -24,143 +24,160 @@ SurveyItem.prototype.buildSurveyItem = function(conatainerId){
+  displayRandomItems();
 }
 
-var chartClicks = itemObjects.map(function(item) {
-  return item.numberOfClicks;
-});
-var chartTimesShown = itemObjects.map(function(item) {
-  return item.numberOfTimesShown;
-});
-var chartNames = itemObjects.map(function(item) {
-  return item.itemName;
-});
-var chartClicksColor = itemObjects.map(function(item) {
-  return '#ad65c8';
-});
-var chartShownColor = itemObjects.map(function(item) {
-  return '#fbd7fb';
-});
-
-var chartLoc = document.getElementById('clickResults');
+function buildAPrettyChart(){
+  var chartClicks = itemObjects.map(function(item) {
+    return item.numberOfClicks;
+  });
+  var chartTimesShown = itemObjects.map(function(item) {
+    return item.numberOfTimesShown;
+  });
+  var chartNames = itemObjects.map(function(item) {
+    return item.itemName;
+  });
+  var chartClicksColor = itemObjects.map(function(item) {
+    return '#ad65c8';
+  });
+  var chartShownColor = itemObjects.map(function(item) {
+    return '#fbd7fb';
+  });
 
-var resultsChart = new Chart(chartLoc, {
-  type: 'horizontalBar',
-  data: {
-    labels: chartNames,
-    datasets: [{
-      label: '# of Clicks',
-      data: chartClicks,
-      backgroundColor: chartClicksColor,
-    },
-    {
-      label: '# of Times Shown',
-      data: chartTimesShown,
-      backgroundColor: chartShownColor,
-    }],
-  },
-  options: {
-    title:{
-      display: true,
-      text: 'Results'
-    },
-    responsive: true,
-    maintainAspectRatio: false,
-    scales: {
-      xAxes: [{
-        stacked: true,
+  var chartLoc = document.getElementById('clickResults');
+  var resultsChart = new Chart(chartLoc, {
+    type: 'horizontalBar',
+    data: {
+      labels: chartNames,
+      datasets: [{
+        label: '# of Clicks',
+        data: chartClicks,
+        backgroundColor: chartClicksColor,
+      },
+      {
+        label: '# of Times Shown',
+        data: chartTimesShown,
+        backgroundColor: chartShownColor,
       }],
-      yAxes: [{
-        stacked: true,
-      }]
+    },
+    options: {
+      title:{
+        display: true,
+        text: 'Results'
+      },
+      responsive: true,
+      maintainAspectRatio: false,
+      scales: {
+        xAxes: [{
+          stacked: true,
+        }],
+        yAxes: [{
+          stacked: true,
+        }]
+      }
     }
-  }
-});
+  });
+}
 
-document.getElementById('surveyButton').addEventListener('click',surveyButtonClick);
-document.getElementsByClassName('surveyItemContainer')[0].addEventListener('click', getRandomItems);
-document.getElementsByClassName('surveyItemContainer')[1].addEventListener('click', getRandomItems);
-document.getElementsByClassName('surveyItemContainer')[2].addEventListener('click', getRandomItems);
+document.getElementById('surveyButton').addEventListener('click',surveyStartButtonClick);
+document.getElementsByClassName('surveyItemContainer')[0].addEventListener('click', function(event){
+  surveyItemContainerClick(event);
+});
```

Comment:
When you find yourself doing something like this...

```javascript
addEventListener('click', function(event){
  surveyItemContainerClick(event);
})
```

...you can almost always pass the function name directly:

```
addEventListener('click', surveyItemContainerClick(event));
```